registry/handlers/app.go

```diff
@@ -458,11 +452,19 @@ func (app *App) authorized(w http.ResponseWriter, r *http.Request, context *Cont
 	if err != nil {
 		switch err := err.(type) {
 		case auth.Challenge:
+			// Since err.ServeHTTP will set the HTTP status code for us
```

Comment:
Wasn't this already present? Clean up the comment, as a note, if you think its necessary:

```
NOTE(duglin): yada yado
```